plugins/modules/ovirt_disk.py

```diff
@@ -459,9 +460,13 @@ def transfer(connection, module, direction, transfer_func):
         while transfer.phase in [
             otypes.ImageTransferPhase.TRANSFERRING,
             otypes.ImageTransferPhase.FINALIZING_SUCCESS,
+            otypes.ImageTransferPhase.FINISHED_SUCCESS
         ]:
             time.sleep(module.params['poll_interval'])
-            transfer = transfer_service.get()
+            try:
```

Comment:
Looks like the code is indented to the right 1 tab too much?
Lines:
465-469
476-478